<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #212c3d;
}</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>580</width>
     <height>340</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 10px;
    font-size: 14px;
    height: 60px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #3498db;
    border: 1px solid #1f2b37;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    width: 80px;
    height: 80px;
}
</string>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xMaxLine">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>151</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 5px;
    font-size: 12px;
    height: 60px;
}</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>66</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    height: 60px;
}</string>
   </property>
   <property name="text">
    <string>xMax</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>66</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    height: 60px;
}</string>
   </property>
   <property name="text">
    <string>yMax</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yMaxLine">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>151</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 5px;
    font-size: 12px;
    height: 60px;
}</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>66</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    height: 60px;
}</string>
   </property>
   <property name="text">
    <string>xMin</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xMinLine">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>151</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 5px;
    font-size: 12px;
    height: 60px;
}</string>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yMinLine">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>151</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 5px;
    font-size: 12px;
    height: 60px;
}</string>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>66</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #3a4e63;
    border: 1px solid #2c3e50;
    color: #ecf0f1;
    padding: 10px;
    font-size: 12px;
    height: 60px;
}</string>
   </property>
   <property name="text">
    <string>yMin</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
